python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -117,7 +121,7 @@ def __init__(self, entity_model, search_label, allow_task_creation, extra_fields
                 self._ui.entity_tree.setModel(filter_model)
 
                 # connect up the filter controls:
-                self._ui.search_ctrl.search_edited.connect(self._on_search_changed)
+                self._ui.search_ctrl.search_changed.connect(self._on_search_changed)
```

Comment:
I wonder if we need some sort of visual indication in the UI that the search will only start when you hit enter now.

I think currently the placeholder says "Search..."

Maybe we should update it to something "Type something and hit enter to search..." or something.